account_invoice_supplierinfo_update/views/account_invoice_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_account_invoice_form" model="ir.ui.view">
+        <field name="model">account.invoice</field>
+        <field name="inherit_id" ref="account.invoice_supplier_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_invoice_open']" position="after">
+                <button name="check_supplierinfo" string="Check Supplier Infos"
+                        help="Check if the invoice informations are similar from the supplier informations"
```

Comment:
information (never plural)
Why define the help message here: wouldnot it be better at the model definition?